src/api/app/models/user.rb

```diff
@@ -338,6 +338,10 @@ def state_transition_allowed?(from, to)
     end
   end
 
+  def cloud_configurations?
+    !!ec2_configuration
```

Comment:
Don't do that (I'm pretty sure rubocop will complain)
Use instead ``ec2_configuration.present?``

Reply:
Why?

Reply:
Double bang does a conversion to a boolean, it obscures your intention, you want to check if it exists so use present!
https://github.com/bbatsov/ruby-style-guide#no-bang-bang